src/databricks/labs/remorph/reconcile/recon_config.py

```diff
@@ -117,8 +117,10 @@ def to_lower_case(input_list: list[str]) -> list[str]:
 
 @dataclass
 class Table:
+    # pylint: disable=too-many-instance-attributes
```

Comment:
We need to ammend pylint in pyproject.toml for this

Reply:
Done